crates/zk-por-cli/src/main.rs

```diff
@@ -39,34 +40,55 @@ pub enum ZkPorCommitCommands {
     },
 }
 
-impl Execute for ZkPorCommitCommands {
+impl Execute for Option<ZkPorCommitCommands> {
     fn execute(&self) -> std::result::Result<(), PoRError> {
         match self {
-            ZkPorCommitCommands::Prove { cfg_path, output_path } => {
+            Some(ZkPorCommitCommands::Prove { cfg_path, output_path }) => {
                 let cfg = zk_por_core::config::ProverConfig::load(&cfg_path)
                     .map_err(|e| PoRError::ConfigError(e))?;
                 let prover_cfg = cfg.try_deserialize().unwrap();
-                let output_file = PathBuf::from_str(&output_path).unwrap();
-                prove(prover_cfg, output_file)
+                let output_path = PathBuf::from_str(&output_path).unwrap();
+                prove(prover_cfg, output_path)
             }
 
-            ZkPorCommitCommands::VerifyGlobal { proof_path: global_proof_path } => {
+            Some(ZkPorCommitCommands::VerifyGlobal { proof_path: global_proof_path }) => {
                 let global_proof_path = PathBuf::from_str(&global_proof_path).unwrap();
-                verify_global(global_proof_path)
+                verify_global(global_proof_path, true)
             }
 
-            ZkPorCommitCommands::VerifyUser { global_proof_path, user_proof_path_pattern } => {
+            Some(ZkPorCommitCommands::VerifyUser {
+                global_proof_path,
+                user_proof_path_pattern,
+            }) => {
                 let global_proof_path = PathBuf::from_str(&global_proof_path).unwrap();
-                verify_user(global_proof_path, user_proof_path_pattern)
+                verify_user(global_proof_path, user_proof_path_pattern, true)
+            }
+
+            None => {
+                println!("============Validation started============");
```

Comment:
Rename validation to verification.

Reply:
This is part of requirement from production team to make the output as identical to [v1](https://www.okx.com/proof-of-reserves)